### ISSUE TITLE
Include translation project in installer

### ISSUE THREAD

**Issue by andersmelander** [andersmelander/better-translation-manager#8]
**[Original report](https://bitbucket.org/anders_melander/better-translation-manager/issues/8) by Anders Melander (Bitbucket: [anders_melander](https://bitbucket.org/anders_melander), ).**

----------------------------------------

In order to enable user translation of BTM, the required files should be made available for install as an optional component in the installer. The files include:

* amTranslationManager.xlat
* amTranslationManager.drc

The drc file is not needed for translation but is required if the user is to generate resource modules. 

---

Depends on issue #7
